fix docs about release (#512)
kyma-project/community: guidelines/releases-guidelines/02-kyma-release-process.md

   * create a GitHub release and trigger documentation update on the official Kyma website
   * trigger provisioning of the cluster from the created release. Use the cluster to test the release candidate.

>    > **CAUTION**: The cluster is automatically generated for you and will be removed after 7 days.

Reviewer: ```suggestion
   > **CAUTION**: The cluster is automatically generated for you, and it is automatically removed after 7 days.
```

Author: Yes :)

Reviewer: Will you apply the comment then? :)

>    > **CAUTION**: The cluster is automatically generated for you and will be removed after 7 days.

Reviewer: ```suggestion
   > **CAUTION**: The cluster is automatically generated for you, and it is automatically removed after 7 days.
```
